src/arrayMethodSort.js

```diff
@@ -4,8 +4,56 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
     // write code here
+    let tempFunction = compareFunction;
+
+    if (typeof compareFunction !== 'function') {
+      tempFunction = function (a, b) {
+        if (String(a) < String(b)) {
+          return -1;
+        }
+
+        if (String(a) > String(b)) {
+          return 1;
+        }
+
+        return 0;
+      };
+    }
+
```

Comment:
and remove this logic

Reply:
add this logic to the argument and remove it from body